src/HyperFormula.ts

```diff
@@ -2837,7 +2837,7 @@ export class HyperFormula implements TypedEmitter {
   }
 
   /**
-   * Returns all addresses and ranges whose computation depends on input address or range provided.
+   * Returns all the out-neighbours in the [dependency graph](../../guide/dependency-graph.md) for a given cell address or range. [Read more](../../guide/dependency-graph.md#getcelldependents).
```

Comment:
The first link (`/guide/dependency-graph.md`) serves a bit different purpose. It offers an explanation for the users that are not familiar with the term `dependency graph` so it's better to have the whole page as a target instead of the subsection on a specific function.

I know we don't have such links but I think we should use them more often. The guides usually contain more comprehensive description of a feature and, more importantly, give the usage context to the API method. A while ago we talked about not having enough cross-links between API reference and the guides. I could change the label `Read more` to something else but I'd rather keep the link.

Reply:
Ok, then please change the link text, according to the typical advice about avoiding simple "Read more" links:
- https://www.nngroup.com/articles/learn-more-links/
- https://blog.prototypr.io/youre-not-still-using-read-more-are-you-16466ae326e7

Reply:
When I thought about it, I decided that it makes much more sense to have the full description of the method in the API reference so there is no need for a "Read more" link. I moved the full description to the API reference and in the guide I left only the link to it. WDYT?

Reply:
I agree it is better that way, thanks!